composer.json

```
@@ -25,18 +25,21 @@
 		"psalm:update-baseline": "psalm.phar --threads=1 --update-baseline",
 		"psalm:clear": "psalm.phar --clear-cache && psalm --clear-global-cache",
 		"psalm:fix": "psalm.phar --alter --issues=InvalidReturnType,InvalidNullableReturnType,MissingParamType,InvalidFalsableReturnType",
-		"test:unit": "vendor/bin/phpunit -c tests/phpunit.xml --color --fail-on-warning --fail-on-risky"
+		"test:unit": "vendor/bin/phpunit -c tests/phpunit.xml --color --fail-on-warning --fail-on-risky",
```

Should be enough, no?

```suggestion
		"test:unit": "phpunit -c tests/phpunit.xml --color --fail-on-warning --fail-on-risky",
```